packages/hoppscotch-common/src/helpers/backend/helpers.ts

```diff
@@ -146,10 +223,26 @@ export const teamCollToHoppRESTColl = (
  * @param teamID - ID of the team
  * @returns Either of the JSON string of the collection or the error
  */
-export const getTeamCollectionJSON = async (teamID: string) =>
-  await runGQLQuery({
+export const getTeamCollectionJSON = async (teamID: string) => {
+  const data = await runGQLQuery({
     query: ExportAsJsonDocument,
     variables: {
       teamID,
     },
   })
+
+  if (E.isLeft(data)) {
+    return E.left(data.left.error.toString())
+  }
+
+  const collections = JSON.parse(data.right.exportCollectionsToJSON)
+
+  if (!collections.length) {
+    const t = getI18n()
+
+    return E.left(t("error.no_collections_to_export"))
+  }
+
+  const hoppCollections = collections.map(teamCollectionJSONToHoppRESTColl)
+  return E.right(JSON.stringify(hoppCollections))
+}
```

Comment:
Previously, bulk export of collections resulted in a format not conforming to `HoppCollection` and hence a re-import resulting in the collection level properties (authorization/headers) missing and the CLI rejecting it (An issue mentioned as patched above). On this front, the `getTeamCollectionJSON()` helper function has been updated to parse the JSON string, transform each entry and return a stringified representation.

Also, the case where there are no collections to export is handled at source returned as a `left` case.